components/net_traits/request.rs

```diff
@@ -178,7 +176,7 @@ impl Default for RequestInit {
             use_cors_preflight: false,
             credentials_mode: CredentialsMode::Omit,
             use_url_credentials: false,
-            origin: ServoUrl::parse("about:blank").unwrap(),
+            origin: ServoUrl::parse("about:blank").unwrap().origin(),
```

Comment:
Opaque origins are generated implicitly by servo, whenever `url.origin()` is called in a case like `about:blank` where the spec calls for a new opaque origin. It doesn't matter a huge amount, this is just the implementation of `default`.